siriuspy/siriuspy/devices/pstesters.py

```diff
@@ -462,7 +462,7 @@ def check_opmode(self, state):
 
     def check_intlk(self):
         """Check interlocks."""
-        return self['AlarmsAmp-Mon'] == 0
+        return self['AlarmsAmp-Mon'] == 0 and self['AlarmsAmpLtc-Mon'] == 0
```

Comment:
Sorry, I just noticed this is only in PS test and not in diag... I don't think there is a case where the instant PV will be different from 0 but the latch won't be.

I think we should remove the last commit, otherwise it might be confusing for someone who opens up the code.